docs/tutorials/google/visualizing_calibration_metrics.ipynb

```diff
@@ -155,102 +158,455 @@
     "print(\"Successful authentication to Google Cloud.\")"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "cellView": "form",
+    "id": "1Oc5MnAWSEkH"
+   },
+   "outputs": [],
+   "source": [
+    "# @title Quantum Engine Configuration\n",
+    "# @markdown Replace the `PROCESSOR_ID` and `YOUR_PROJECT_ID` with your details.\n",
+    "PROCESSOR_ID = 'weber' #@param {type: 'string'}\n",
```

Comment:
Pretty sure `weber` has not been made available yet - I recommend leaving this blank, or using a different default.

Reply:
done